### DISCUSSION TITLE
How to install Pytorch-Lightning on Conda with `extras` dependencies?

### DISCUSSION THREAD

**Discussion by FeryET** [Lightning-AI/pytorch-lightning#10723] [General]
I was reading [this](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_cli.html), and apparently to get `LightningCLI` you need to install lightning with [extras] in pip. But I cannot find how to do this in conda. Is there anyway to do this, or should I use pip instead?

**Comment by kaushikb11** [comment]
Seems like the issue hasn't been resolved yet. https://github.com/conda/conda/issues/2984

cc: @Borda
